compiler/rustc_lint/src/internal.rs

```diff
@@ -267,3 +267,42 @@ impl EarlyLintPass for LintPassImpl {
         }
     }
 }
+
+declare_tool_lint! {
+    pub rustc::EXISTING_DOC_KEYWORD,
+    Allow,
+    "Check that documented keywords in std and core actually exist",
+    report_in_external_macro: true
+}
+
+declare_lint_pass!(ExistingDocKeyword => [EXISTING_DOC_KEYWORD]);
+
+fn is_doc_keyword(s: Symbol) -> bool {
+    s <= kw::Union
```

Comment:
Can you use `Symbol::is_doc_keyword` here?

Reply:
It's been removed. :)

Reply:
:scream: